src/make_ntuples.c

```diff
@@ -62,14 +62,26 @@ double get_tof(REC_Scintillator rsci, REC_Calorimeter  rcal, int pindex) {
     return tof;
 }
 
-int run(char * in_filename, bool debug, int nevn, int run_no, double beam_E) {
-    // Extract sampling fraction parameters.
-    double sf_params[NSECTORS][SF_NPARAMS][2];
-    if (get_sf_params(Form("../data/sf_params_%06d.root", run_no), sf_params)) return 8;
+int run(char * in_filename, bool use_simul,bool debug, int nevn, int run_no, double beam_E) {
+    double sf_params[NSECTORS][SF_NPARAMS][2] = {{{0.246,0.0140},{0.985,1.},{0.083,0.},{-0.031,0.}},
+                                                 {{0.255,0.0150},{0.985,1.},{0.043,0.},{-0.024,0.}},
+                                                 {{0.245,0.0130},{0.985,1.},{0.067,0.},{-0.025,0.}},
+                                                 {{0.245,0.0140},{0.985,1.},{0.048,0.},{-0.016,0.}},
+                                                 {{0.252,0.0130},{0.985,1.},{0.007,0.},{-0.007,0.}},
+                                                 {{0.242,0.0140},{0.985,1.},{0.095,0.},{-0.038,0.}}};
```

Comment:
What is this? Please add these numbers to a pre-prepared file in the data directory instead of hardcoding them.

Reply:
Added this values to process the simulations. Moved the values to constants.C.
Also, added a variation of get_sf_params() that works for simuls and mantains the format of obtaining sf params.

Reply:
Forgot to mention this: These are sampling fraction parameters obtained from the CCDB.

Reply:
Could you move them to the data folder? You can add an exception to the .gitignore there to allow this. It's a much cleaner alternative and mimics the behaviour used for data.

Reply:
you mean to add a piece of code with these values on the data folder?

Reply:
When you run the extract_sf program, you get a simple text file similar to this one:

    0.26300 0.98500 -0.03600 0.00200 0.01660 1.0 0.0 0.0
    0.24022 1.00000 -0.01833 0.00050 0.01825 1.0 0.0 0.0
    0.23976 1.00000 -0.01600 0.00070 0.02249 1.0 0.0 0.0
    0.23774 1.00000 -0.01600 0.00070 0.01816 1.0 0.0 0.0
    0.25118 1.00000 -0.02615 0.00070 0.01735 1.0 0.0 0.0
    0.24044 1.00000 -0.01921 0.00050 0.01817 1.0 0.0 0.0
    0.24210 1.00000 -0.01713 0.00066 0.01646 1.0 0.0 0.0

where each line is a sector, the first four params are sf1 through sf4, and the following params are sfs1 through sfs4. This is the same format you can see on the [CCDB](https://clasweb.jlab.org/cgi-bin/ccdb/show_request?request=/calibration/eb/electron_sf:0:default:2018-06-11_19-11-20).

You can manually write down a file with this same format containing the values from your hardcoded array, and the program will read it no problem.

Reply:
done!